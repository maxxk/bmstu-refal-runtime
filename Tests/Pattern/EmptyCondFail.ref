$ENTRY Go
{
	= <Prout <Func asdasd>> 
}

Func
{
	= 'Empty Condition'
}
