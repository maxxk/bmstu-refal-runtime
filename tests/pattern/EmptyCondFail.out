[VM]: Func call failed!
Func call term:
<Func asdasd >
Field of view:
<Prout <Func asdasd >>
